operator/iamkeycontroller/create.go

```diff
@@ -10,11 +10,10 @@ import (
 	"github.com/crossplane/crossplane-runtime/pkg/resource"
 	v2 "github.com/exoscale/egoscale/v2"
```

Comment:
I wasn't sure what `v2` is, so I'd like to see it a bit more clearly and to be consistent with other usages.
```suggestion
	exoscalesdk "github.com/exoscale/egoscale/v2"
```

Reply:
Good observation, I updated it.